mlir/utils/performance/tuningRunner.py

```diff
@@ -266,7 +269,17 @@ def tune_mlir_kernels(configs, conf_class, paths: Paths, options: Options):
                                                    stdin=kernel_gen.stdout,
                                                    stdout=subprocess.PIPE,
                                                    stderr=subprocess.PIPE)
-                    kernel_gen.stdout.close()
+                    # Wait for both processes to finish.
+                    tuning_loop_stdout, _ = tuning_loop.communicate()
```

Comment:
Previously, the output would be streamed inside `get_winning_config`. Now we wait for the process to finish and handle the entire output. Not a big deal I think, but streaming did give us a better look into how things are progressing in real time.